src/test/java/teammates/test/cases/action/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -695,6 +703,97 @@ public void testExecuteAndPostProcess() throws Exception {
                 r.getDestinationWithParams());
         assertNotNull(frDb.getFeedbackResponse(fq.getId(), fr.giver, fr.recipient));
 
+        ______TS("Unsuccessful case: const sum: over maximum points");
+        String maxValueQn1 = "90";
+
+        submissionParams = new String[] {
+                Const.ParamsNames.FEEDBACK_QUESTION_RESPONSETOTAL + "-1", "1",
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID + "-1-0", fr3.getId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, fr3.feedbackSessionName,
+                Const.ParamsNames.COURSE_ID, fr3.courseId,
+                Const.ParamsNames.FEEDBACK_QUESTION_ID + "-1", fr3.feedbackQuestionId,
+                Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-1-0", fr3.recipient,
+                Const.ParamsNames.FEEDBACK_QUESTION_TYPE + "-1", fr3.feedbackQuestionType.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-1-0", "91",
+                Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-1-0", "9",
```

Comment:
It seems like u missed out my last review comment.

Reply:
I've changed the feedback response to fr3. It's a new feedback response I created in the json, corresponding to a new question where there's no `MIN_CONSTRAINT`. It's impossible to test both constraints within 1 question. So I created a new question without `MIN_CONSTRAINT`, which means `9` does not violate `MIN_CONSTRAINT and I'm only testing MAX_CONSTRAINT in this case.

This answers the 2 other comments above.